model.py

```diff
@@ -223,7 +221,7 @@ def precompute_freqs_cis(
     freqs = torch.outer(t, freqs)
     freqs_cis = torch.polar(torch.ones_like(freqs), freqs)
     cache = torch.stack([freqs_cis.real, freqs_cis.imag], dim=-1)
-    return cache.to(dtype=torch.bfloat16)
+    return cache.to(dtype=torch.float16)
```

Comment:
I assume this needs to become a parameter

Reply:
yeah, will make this a parameter before landing